golink.go

```diff
@@ -39,7 +39,17 @@ import (
 	"tailscale.com/util/dnsname"
 )
 
-const defaultHostname = "go"
+const (
+	defaultHostname = "go"
+
+	// Used as a placeholder short name for generating the XSRF defense token,
+	// when creating new links.
+	newShortName = ".new"
+
+	// JavaScript in browsers cannot set this header, per the Fetch Spec
```

Comment:
Maybe also say how we're using it?

```go
// If the caller sends this header set to a non-empty value, we will allow them
// to make the call even without an XSRF token. JavaScript in browser cannot set
// this header, per the [Fetch Spec].
//
// [Fetch Spec]: https://fetch.spec.whatwg.org
```

perhaps.

Reply:
Done!